iOS/MSMapsModules/.gitignore

```diff
@@ -0,0 +1,31 @@
+.DS_STORE
```

Comment:
Add .DS_Store to .gitignore at the repo root and remove the two added .DS_Store files.

Reply:
This entry isn't needed anymore since it's in the root .gitignore.